docs/permissions.md

```diff
@@ -9,3 +9,10 @@ Additionally, the OpenSAFELY framework requires a user to be a member of the `op
 While your repository is private, you may need to add new collaborators who are not named in your original application. Any new collaborators have the same information governance obligations as the original study authors, and must therefore sign the same access agreements; please send any such requests to [team@opensafely.org](mailto:team@opensafely.org).
 
 
+## Project Member Management
+
```

Comment:
We could add some preamble here. How about something like this

"When they are approved to use the platform, every OpenSAFELY user is assigned roles that define what they are permitted to do. By default, most researchers using the platform will have `X` role, which permits them to X Y Z.  Additionally, users with the `ProjectCoordinator` role can.... <etc>.